src/core/plugins/oas3/components/request-body.jsx

```diff
@@ -150,39 +150,49 @@ const RequestBody = ({
       <table>
         <tbody>
           {
-            Map.isMap(bodyProperties) && bodyProperties.entrySeq().map(([key, prop]) => {
-              if (prop.get("readOnly")) return
+            Map.isMap(bodyProperties) && bodyProperties.entrySeq().map(([key, schema]) => {
+              if (schema.get("readOnly")) return
 
-              let commonExt = showCommonExtensions ? getCommonExtensions(prop) : null
+              const oneOf = schema.get("oneOf")
+              const anyOf = schema.get("anyOf")
+
+              if (oneOf || anyOf) {
```

Comment:
Perhaps we could move handling these below this line
```
  let initialValue = fn.getSampleSchema(schema, false, {
    includeWriteOnly: true
  })
```
So that if the parent schema has an example, we would get the initial value from it. We would need to reassign the schema, type, format and description then.